### PR TITLE
Some of the app update QA sheet

### PR DESCRIPTION
**Pull request by phillipm**
Test a single app update that deletes a module and changes the multimedia of a form question. Also tries to download an update in airplane mode.

Anything more complicated would require more fine-grained control over HQ form builder and deploys.

Found the following bug while writing this: https://github.com/dimagi/commcare-android/pull/1461

### CODE REVIEW

File: features/app_update.feature [hunk 1]
```diff
@@ -1,33 +1,56 @@
...
+    # check updated data, including multimedia
+    Then I login with username "user_with_no_data" and password "123"
 
-    Then I select module "first module"
-    And I select form "empty form"
-    Then I see the text "An updated label"
+    Then I press start
+    Then I don't see the text "Module Three"
+    Then I select module "Module One"
+    And I select form "Example 1"
```
Comment (amstone326) [inline features/app_update.feature]: What does the `And` prefix do differently?

File: features/step_definitions/main_structure_steps.rb [hunk 2]
```diff
@@ -113,3 +112,18 @@
     fail("Found %s occurrences of %s; expected none" % [count, text])
   end
 end
+
+Then (/^I see at least one element of type "([^\"]*)"$/) do |element|
```
Comment (amstone326) [inline features/step_definitions/main_structure_steps.rb]: could you move these 3 step definitions to `inspection_steps.rb`

File: features/app_update.feature (hunk 1 above)
Comment (phillipm) [inline features/app_update.feature]: It is just a way to link commands together. Don't think it really has any use, as far as I can tell...

File: features/step_definitions/main_structure_steps.rb (hunk 2 above)
Comment (phillipm) [inline features/step_definitions/main_structure_steps.rb]: 👍

**Comment by amstone326** [comment]
@phillipm Good to merge this one once merge conflicts are resolved

**Comment by phillipm** [comment]
@amstone326 conflicts resolved

**Pull request merged**